src/HoloColorPicker.js

```diff
@@ -179,7 +179,7 @@ export class HoloColorPicker extends React.PureComponent {
               activeOpacity={0.7}
             />
             }
-            {!oldColor &&
+            {!oldColor && this.props.hideOldColor === false &&
```

Comment:
I believe here the prop does not make sense, since it has the setting `hideOldColor` to true has the same effect as not setting `oldColor`.

And missing default prop with === would change the default behaviour, however I believe it shouldn't be here at all (if I am not mistaken)